docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+
+services:
+  db:
+    image: mongo
+    container_name: mongo_db
+    ports:
+      - "27017:27017"
+    volumes:
+      - ${PWD}/db:/data/db
```

Comment:
odkial sa berie ta premenna **PWD** ? tu musis mat nejak externa nasetovanu? Co mi chudaci co vyvyjame na windows?

Reply:
Je to aktualny adresar... ak to pustis v powershelli tak by to malo fungovat. 
Inak odporucam wsl :)